src/main/java/org/commcare/formplayer/beans/menus/BaseResponseBean.java

```diff
@@ -47,11 +50,15 @@ public void setNotification(NotificationMessage notification) {
         this.notification = notification;
     }
 
-    public boolean isClearSession() {
+    public void shouldAutoSubmit(boolean shouldAutoSubmit) {
```

Comment:
I believe you need to add a getter (getShouldAutoSubmit) to the class or provide a method annotation for the field to be included in the JSON payload for the bean.